ui/lib/nance/index.ts

```diff
@@ -91,3 +91,312 @@ export const TEMPLATE = `\n*Note: Please remove the italicized instructions befo
 | :---- | :---- | :---- | :---- |
 | *Send* | *0* | *ETH* | *TBD* |
 `
+
+export const FINAL_REPORT_TEMPLATE = `
+*The title will be included at the top of the final report*
+
+# **Results**
+
+1. **Objective:**   
```

Comment:
This template is outdated - can you sync with mitchie for the correct markdown template?